src/binder/binder.cpp

```diff
@@ -220,12 +220,16 @@ function::TableFunction Binder::getScanFunction(FileTypeInfo typeInfo, const Rea
             csvConfig.parallel ? ParallelCSVScan::name : SerialCSVScan::name, inputTypes,
             functions);
     } break;
-    case FileType::TURTLE:
     case FileType::UNKNOWN: {
         try {
             func = function::BuiltInFunctionsUtils::matchFunction(clientContext->getTx(),
-                SCAN_JSON_FUNC_NAME, inputTypes, functions);
+                common::stringFormat("{}_SCAN", typeInfo.fileTypeStr), inputTypes, functions);
         } catch (...) {
+            if (typeInfo.fileTypeStr == "") {
```

Comment:
Cover

Reply:
this is already covered by extension test which doesn't show up correctly in codecov